src/jit/morph.cpp

```diff
@@ -8245,6 +8245,8 @@ GenTree* Compiler::fgMorphCall(GenTreeCall* call)
         }
     }
 
+    compCurBB->bbFlags |= BBF_HAS_CALL;
```

Comment:
There are various places above where we return early. If indicating this here is okay, can you comment as to why the above doesn't count as a call?